src/Malinina.Homework.FitnessTracker/Training.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace Malinina.Homework.FitnessTracker
+{
+    /// <summary>
+    /// Training description class.
+    /// </summary>
+    internal class Training
+    {
+        /// <summary>
+        /// Year, month and day of training session.
+        /// </summary>
+        public DateTime DateTraining { get; private set; }
```

Comment:
Properties looks like read-only, so they can be defiled as 

`public DateTime DateTraining { get; }`

Not a big deal but good to know